.vimbundle

```diff
@@ -1,54 +1,74 @@
-hashrocket/vim-hashrocket
-AndrewRadev/splitjoin.vim
-adamlowe/vim-slurper
-jbranchaud/vim-bdubs
-ervandew/supertab
-godlygeek/tabular
-gregsexton/gitv
-heartsentwined/vim-emblem
-jgdavey/tslime.vim
-jgdavey/vim-blockle
-jgdavey/vim-railscasts
-jgdavey/vim-turbux
-jgdavey/vim-weefactor
-kchmck/vim-coffee-script
-leshill/vim-json
-mileszs/ack.vim
-pangloss/vim-javascript
-rondale-sc/vim-spacejam
-slim-template/vim-slim
-therubymug/vim-pyte
-tpope/vim-abolish
-tpope/vim-bundler
-tpope/vim-commentary
-tpope/vim-cucumber
-tpope/vim-endwise
-tpope/vim-eunuch
-tpope/vim-fugitive
-tpope/vim-git
-tpope/vim-haml
-tpope/vim-markdown
-tpope/vim-pathogen
-tpope/vim-projectionist
-tpope/vim-ragtag
-tpope/vim-rails
-tpope/vim-rake
-tpope/vim-repeat
-tpope/vim-rsi
-tpope/vim-sensible
-tpope/vim-sleuth
-tpope/vim-speeddating
-tpope/vim-surround
-tpope/vim-unimpaired
-tpope/vim-vinegar
-tpope/vim-vividchalk
-vim-ruby/vim-ruby
-vim-scripts/bufexplorer.zip
-vim-scripts/bufkill.vim
-wgibbs/vim-irblack
-kana/vim-textobj-user
-nelstrom/vim-textobj-rubyblock
-exu/pgsql.vim
-elixir-lang/vim-elixir
-mxw/vim-jsx
-sbdchd/neoformat
+" vim: ft=vim
+" Vim Plugins via Vim Plug
+" Specify a directory for plugins
+" - For Neovim: ~/.local/share/nvim/plugged
+" - Avoid using standard Vim directory names like 'plugin'
+if &compatible
+  set nocompatible
+end
+
+call plug#begin('~/.vim/bundle')
+
+
+" Make sure you use single quotes
+Plug 'hashrocket/vim-hashrocket'
+Plug 'AndrewRadev/splitjoin.vim'
+Plug 'adamlowe/vim-slurper'
+Plug 'jbranchaud/vim-bdubs'
+Plug 'ervandew/supertab'
+Plug 'godlygeek/tabular'
+Plug 'gregsexton/gitv'
+Plug 'heartsentwined/vim-emblem'
+Plug 'jgdavey/tslime.vim'
+Plug 'jgdavey/vim-blockle'
+Plug 'jgdavey/vim-railscasts'
+Plug 'jgdavey/vim-turbux'
+Plug 'jgdavey/vim-weefactor'
+Plug 'kchmck/vim-coffee-script'
+Plug 'leshill/vim-json'
+Plug 'mileszs/ack.vim'
+Plug 'pangloss/vim-javascript'
+Plug 'rondale-sc/vim-spacejam'
+Plug 'slim-template/vim-slim'
+Plug 'therubymug/vim-pyte'
+Plug 'tpope/vim-abolish'
+Plug 'tpope/vim-bundler'
+Plug 'tpope/vim-commentary'
+Plug 'tpope/vim-cucumber'
+Plug 'tpope/vim-endwise'
+Plug 'tpope/vim-eunuch'
+Plug 'tpope/vim-fugitive'
+Plug 'tpope/vim-git'
+Plug 'tpope/vim-haml'
+Plug 'tpope/vim-markdown'
+Plug 'tpope/vim-pathogen'
```

Comment:
hey @dkarter, is this `vim-pathogen` still necessary?

Reply:
Sharp Eyes @vnegrisolo :eye: :eye:

Reply:
good catch @vnegrisolo - removed.